components/hab/src/command/pkg/uninstall.rs

```diff
@@ -217,7 +217,11 @@ fn do_clean_delete(pkg_root_path: &Path, real_install_path: &Path) -> Result<boo
     // below the pkg_root_path
     match real_install_path.parent() {
         Some(real_install_base) => {
+            // create empty temp directory
             let temp_install_path = temp_package_directory(real_install_path)?;
```

Comment:
When trying to find the `temp_package_directory` function, I noticed that the `use` statements at the top of this module are in separate "chunks" and also mix up a lot of external crate usage with internal usage.

While you're in here, it'd be a great cleanup opportunity!